frontend/src/features/idir/search/components/IDIRCompanySearchResults.tsx

```diff
@@ -158,13 +159,18 @@ export const IDIRCompanySearchResults = memo(
           header: "Company Name",
           enableSorting: true,
           sortingFn: "alphanumeric",
+          minSize: 220,
           Cell: (props: { row: any; cell: any }) => {
+            const isCompanySuspended = props.row.original.isSuspended;
             return (
-              <CustomActionLink
-                onClick={() => onClickCompany(props.row.original)}
-              >
-                {props.row.original.legalName}
-              </CustomActionLink>
+              <>
+                <CustomActionLink
+                  onClick={() => onClickCompany(props.row.original)}
```

Comment:
What exactly is ```props.row.original``` (its type)? Is it the company info?

Reply:
It's guaranteed to never be null/undefined right?

Reply:
Yep the type would be CompanyProfile, I can add this in if you prefer? In terms of this ever being null/undefined I do not believe this is possible since the column definition will only be rendered if there is available CompanyProfile data to begin with

Reply:
Ok good, because if for any reason it's null/undefined, then ```props.row.original.isSuspended``` would yield a runtime error